src/Drupal/DrupalExtension/Context/RawDrupalContext.php

```diff
@@ -603,4 +611,39 @@ public function loggedInWithRole($role) {
     return $this->loggedIn() && $this->getUserManager()->currentUserHasRole($role);
   }
 
+  /**
+   * Create an entity.
+   *
+   * @return object
+   *   The created entity.
+   */
+  public function entityCreate($entity_type, $entity) {
+	if (empty($entity_type)) {
+	        throw new \Exception("Steps must specify an entity type in order to create an entity.");
+	}
+    // We want hooks & cleanup to know the entity type
```

Comment:
Tabs used instead of spaces for indentation, I also saw this in some other places.